autosklearn/evaluation/abstract_evaluator.py

```diff
@@ -338,22 +339,23 @@ def _loss(
         ----------
             y_true
         """
-        scoring_functions = (
-            self.scoring_functions if scoring_functions is None else scoring_functions
-        )
         if not isinstance(self.configuration, Configuration):
-            if scoring_functions:
-                return {self.metric.name: self.metric._worst_possible_result}
-            else:
-                return self.metric._worst_possible_result
-
-        return calculate_loss(
-            y_true,
-            y_hat,
-            self.task_type,
-            self.metric,
-            scoring_functions=scoring_functions,
-        )
+            # Dummy prediction
+            rval = {}
+            for metric in self.scoring_functions if self.scoring_functions else []:
```

Comment:
That's cool, didn't know you could have if statements in for loops like this, nice :)